### PR TITLE
Add data cy to select close icon

### LINKED ISSUE #1999: Add data-cy for single and multi option clear button
Extracted from https://github.com/bigbinary/neeto-fields-nano/issues/514

image

### PR DESCRIPTION
**Pull request by ittyjose**
- Fixes #1999 

**Description**
Added: data cy to close select/multiSelect icon 

**Checklist**

~- [ ] I have made corresponding changes to the documentation.~
~- [ ] I have updated the types definition of modified exports.~
~- [ ] I have verified the functionality in some of the neeto web-apps.~
~- [ ] I have added tests that prove my fix is effective or that my feature works~
- [x] I have added the necessary label (patch/minor/major - If package publish
      is required).

**Reviewers**
@Amaljith-K _a

### CODE REVIEW

File: src/components/Select.jsx [hunk 1]
```diff
@@ -45,7 +45,10 @@ const DropdownIndicator = props => (
 );
 
 const ClearIndicator = props => (
-  <components.ClearIndicator {...props}>
+  <components.ClearIndicator
+    {...props}
+    innerProps={{ ...props.innerProps, ["data-cy"]: "clear-select-indicator" }}
```
Comment (ajmaln) [inline src/components/Select.jsx:50]:
```suggestion
    innerProps={{ ...props.innerProps, "data-cy": "clear-select-indicator" }}
```

**Review by ajmaln — COMMENTED** [review]
@ittyjose _a Please assign to @yatharthvij for review after updating

**Comment by ittyjose** [comment]
Please review @yatharthvij _a

**Review by yatharthvij — APPROVED** [review]
@ittyjose _a LGTM.

**Pull request merged**